Application/Sources/Controllers/Authentication/LoginViewController.swift

```diff
@@ -4,29 +4,35 @@
 //
 
 import AppwiseCore
+import TestHelpers
 import UIKit
 
 final class LoginViewController: UIViewController {
+	@IBOutlet private var loginButton: UIButton!
+
+	override var preferredStatusBarStyle: UIStatusBarStyle {
+		.lightContent
+	}
+
 	override func viewDidLoad() {
 		super.viewDidLoad()
 
+		loginButton.accessibilityIdentifier = AccessibilityIdentifiers.loginButton
```

Comment:
I'm not very fond of setting identifiers like this if it isn't needed. But it's passable here because most apps need a ref to the button to disable it during load.